wakuv2/waku.go

```diff
@@ -1677,13 +1715,10 @@ func (w *Waku) seedBootnodesForDiscV5() {
 			}
 		// If we go online, trigger immediately
 		case <-w.goingOnline:
-			if w.cfg.EnableDiscV5 {
-				if canQuery() {
-					err := w.restartDiscV5()
-					if err != nil {
-						w.logger.Warn("failed to restart discv5", zap.Error(err))
-					}
-
+			if canQuery() {
```

Comment:
same comment as above, we may need to have EnableDiscv5 check to prevent restarting in case of lightclient.